ui/src/views/infra/HostInfo.vue

```diff
@@ -83,15 +83,15 @@
           </div>
         </div>
       </a-list-item>
-      <a-list-item v-if="host.hostha">
+      <a-list-item v-if="host.hostha && host.hostha.haenable">
```

Comment:
Don't think this should be changed so that the user can explicitly see whether ha is enabled or not

Reply:
the `host.hostha.haenable` state is already  shown above, so the `host.hostha.hastate` should only be relevant, if `haenable` is True

Reply:
Ah yes, my bad